src/radical/pilot/agent/rm/fork.py

```diff
@@ -38,7 +38,12 @@ def _configure(self):
         # to preserve the previous behavior (1 node).
         self.cores_per_node = self._cfg.get('cores_per_node', self.requested_cores)
         self.gpus_per_node  = self._cfg.get('gpus_per_node', 0)  # FIXME GPU
-        self.lfs_per_node   = {'path' : self._cfg.get('lfs_path_per_node', None),
+
+        lfs_path = self._cfg.get('lfs_path_per_node', None)
+        if '$' in path:
+            lfs_path = os.path.expandvars(path)
+
+        self.lfs_per_node   = {'path' : lfs_path,
                                'size' : self._cfg.get('lfs_size_per_node', 0)
```

Comment:
Fork'ed CUs will always have the same env as the agent, so this is not needed here.  The problem should only occur on  launch methods which define a hop command, and where that hop command does not forward the agent env.

Reply:
Okay! I will remove it!